app/views/drawings/_form.html.haml

```diff
@@ -17,23 +17,25 @@
         .form-group
           = f.input :country, priority: TOP_COUNTRIES, selected: (@drawing.new_record? ? current_user.country : @drawing.country), label: "Country of Origin", include_blank: "Country picture was drawn in", class: 'form-control'
         .form-group
-          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: false, required: true, prompt: "Subject matter of drawing"
+          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: 'Subject matter of drawing', required: true, selected: "Home / Country of origin"
```

Comment:
I would add a `required: false` to stop the * showing up next to the label. We have complex rules around what is required and not, which are bing dealt with already by the text ("required when marking complete")

The alternative is add a `required: true` to country of origin and description of drawing. So all three of these get an * on their label